dev-docs/source/howto/dbimports.rst

```diff
@@ -0,0 +1,18 @@
+Database Imports
+================
+
+Downloading Database Dump Files
+-------------------------------
```

Comment:
Is this still working? My assumption was that since the new reference db, the data import is not working anymore.